### PR TITLE
feat: Image dimensions update management command

### PR DESCRIPTION
**Pull request by benzkji**
## Description

Basic management command, to insert missing image dimensions.
I've added a test, but can't find instructions on how to setup local tests.

Also, I've only seen conventional commits just now...so...

* [x] I have opened this pull request against ``master``
* [x] I have added or modified the tests when changing logic
* [ ] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-filer/blob/master/docs/development.rst#contributing) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Comment by benzkji** [comment]
I guess I am doing too much work here. It would probably better to call some helper function of filer? Change a field, call save(), and thus triggering calculation of _width and _height (and possibly other meta fields).

Also, my test fails, probalby because the provided JPG is not a real one?

File: filer/management/commands/filer_check.py [hunk 1]
```diff
@@ -112,3 +121,30 @@ def walk(prefix):
...
+            try:
+                imgfile = image.file.file
+            except ValueError:
+                imgfile = image.file_ptr.file
+            imgfile.seek(0)
+            if image.file.name.endswith('.svg'):
+                image._width, image._height = VILImage.load(imgfile).size
+            else:
+                pil_image = PILImage.open(imgfile)
+                self._width, self._height = pil_image.size
```
Comment (fsbraun) [inline filer/management/commands/filer_check.py:147]:
```suggestion
                image._width, image._height = pil_image.size
```

I am not sure if `self` is correct here...

File: filer/management/commands/filer_check.py [hunk 2]
```diff
@@ -112,3 +121,30 @@ def walk(prefix):
...
+                self._transparent = easy_thumbnails.utils.is_transparent(pil_image)
```
Comment (fsbraun) [inline filer/management/commands/filer_check.py:148]:
```suggestion
                image._transparent = easy_thumbnails.utils.is_transparent(pil_image)
```

Same here..

**Comment by fsbraun** [comment]
@benzkji I would be fine if you caught the `PIL.UnidentifiedImageError`, and potentially any `FileNotFound` error in the management command. They indicate a inconsistent state: corrupted or missing files.

**Comment by benzkji** [comment]
probably last thing: test with SVG, and "corrupt" SVG.

**Comment by fsbraun** [comment]
@benzkji I think, also a SVG-based test would be great. Here's a way to easily test SVGs: 

https://github.com/benzkji/django-filer/blob/6aa899527585d6d352bb0c008cf451791e168a4c/tests/test_validation.py#L26-L30

**Review by fsbraun — APPROVED** [review]
Great work, @benzkji ! Thanks so much!

**Pull request merged**